Source/Layout/ASLayout.mm

```diff
@@ -373,22 +374,6 @@ - (NSString *)_recursiveDescriptionForLayout:(ASLayout *)layout level:(NSUIntege
   return description;
 }
 
-- (BOOL)isEqual:(id)object
-{
-  if (self == object) {
-    return YES;
-  }
-
-  if (ASLayout *layout = ASDynamicCast(object, ASLayout)) {
-    return CGSizeEqualToSize(self.size, layout.size)
-    && ((ASPointIsNull(self.position) && ASPointIsNull(layout.position))
-      || CGPointEqualToPoint(self.position, layout.position))
-    && self.layoutElement == layout.layoutElement
-    && ASObjectIsEqual(self.sublayouts, layout.sublayouts);
-  }
-  return NO;
-}
-
```

Comment:
Oh wow. Very very scary that this never warned of duplicate method implementation.